firefox-ios/nimbus-features/messaging/messaging.fml.yaml

```diff
@@ -204,3 +225,13 @@ enums:
         description: The next eligible message should be shown.
       show-none:
         description: The surface should show no message.
+
+  TargetFeature:
```

Comment:
@tom instead of having both the target feature and icon, it might be more simpler to just have this single enum here and then configure the icon in the codebase. Makes the experiment recipe simpler and since we have to declare a feature each time anyways, we can declare the icon at the same time, so don't see much benefit in making it configurable. It will also be less error prone. Let me know what you think!

Reply:
Agreed, I was thinking similar after our conversation 😄

Reply:
great!